build/dev-setup.sh

```diff
@@ -72,6 +72,14 @@ npm link
 npm run build
 [ $? -eq 0 ] || exit $?
 
+cd $node_root/provisioning/transport/http
+echo "\n-- Creating links for `pwd` --"
+npm link azure-iot-common
+npm link azure-iot-http-base
```

Comment:
>http [](start = 19, length = 4)

not sure if it matters but in other packages/files is linked http-base comes before common